src/main/java/spoon/reflect/CtModel.java

```diff
@@ -40,6 +42,16 @@
 	/** returns all packages of the model */
 	Collection<CtPackage> getAllPackages();
 
+	/**
+	 * returns the unnamed module
```

Comment:
returns the unnamed module. Warning: if there are other modules, they are not contained in that unnamed root module.